pkg/types/spanpercentiletypes/spanpercentile.go

```diff
@@ -0,0 +1,43 @@
+package spanpercentiletypes
+
+import (
+	"github.com/SigNoz/signoz/pkg/errors"
+)
+
+type SpanPercentileRequest struct {
+	DurationNano       int64             `json:"span_duration"`
```

Comment:
```suggestion
	DurationNano       int64             `json:"spanDuration"`
```

PascalCase for requests. See other request response JSONs